pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitCreationStatusTest.java

```diff
@@ -66,68 +87,109 @@ public static void deleteJuniperInstance() {
 
-        try {
-            createAndAssertNonPepperCreation(juniperTestKit);
-
-            KitResponse kitResponse = nonPepperStatusKitService.getKitsByStudyName(instanceGuid);
-            Assert.assertNotNull(kitResponse);
-            Assert.assertNotNull(kitResponse.getKits());
-            Assert.assertNotEquals(0, kitResponse.getKits().size());
-            Assert.assertNotNull(
-                    kitResponse.getKits().stream().filter(kitStatus -> kitStatus.getJuniperKitId().equals("JuniperTestKitId_" + rand))
-                            .findAny());
-        } finally {
-            createdKitIds.add(juniperTestKit.getJuniperKitId());
-        }
     }
 
     @Test
     public void testStatusByJuniperKitId() {
         int rand = new Random().nextInt() & Integer.MAX_VALUE;
         JuniperKitRequest juniperTestKit = generateJuniperKit(rand);
-        try {
-            createAndAssertNonPepperCreation(juniperTestKit);
+        createAndAssertNonPepperCreation(juniperTestKit);
+        KitResponse kitResponse = nonPepperStatusKitService.getKitsBasedOnJuniperKitId(juniperTestKit.getJuniperKitId());
 
-            KitResponse kitResponse = nonPepperStatusKitService.getKitsBasedOnJuniperKitId(juniperTestKit.getJuniperKitId());
-
-            assertStatusKitResponse(kitResponse, juniperTestKit, rand);
-        } finally {
-            createdKitIds.add(juniperTestKit.getJuniperKitId());
-        }
+        assertStatusKitResponse(kitResponse, juniperTestKit, rand, KitCurrentStatus.KIT_WITHOUT_LABEL.getValue());
     }
 
     @Test
-    public void testStatusByKitIdTest() {
+    public void testStatusByParticipantIdTest() {
         int rand = new Random().nextInt() & Integer.MAX_VALUE;
         JuniperKitRequest juniperTestKit = generateJuniperKit(rand);
-        try {
-            createAndAssertNonPepperCreation(juniperTestKit);
-            KitResponse kitResponse = nonPepperStatusKitService.getKitsBasedOnJuniperKitId(juniperTestKit.getJuniperKitId());
-            assertStatusKitResponse(kitResponse, juniperTestKit, rand);
+        createAndAssertNonPepperCreation(juniperTestKit);
+        KitResponse kitResponse = nonPepperStatusKitService.getKitsBasedOnParticipantId(juniperTestKit.getJuniperParticipantID());
+        assertStatusKitResponse(kitResponse, juniperTestKit, rand, KitCurrentStatus.KIT_WITHOUT_LABEL.getValue());
 
-        } finally {
-            createdKitIds.add(juniperTestKit.getJuniperKitId());
-        }
     }
 
     private void createAndAssertNonPepperCreation(JuniperKitRequest juniperTestKit) {
-        List<KitRequestShipping> oldkits = KitRequestShipping.getKitRequestsByRealm(instanceName, "overview", salivaKitType);
-        KitResponse kitResponse = nonPepperKitCreationService.createNonPepperKit(juniperTestKit, salivaKitType, mockEasyPostUtil, ddpInstance);
+        createdKitIds.add(juniperTestKit.getJuniperKitId());
```

Comment:
I need to call `generateJuniperKit` (to be renamed to `generateJuniperKitRequest` separately because the returned value of that is useful outside of the scope of this method to my tests

Reply:
It still feels like the parameter name here does not match the parameter type, and the method name still misses the fundamental purpose of this method, which is to create a kit for the caller. Perhaps a nit, but I do not think the purpose is clear at all, so if you like the method name, perhaps just some JavaDoc explaining that this method creates a kit? All the other stuff in the method seems secondary.